### DISCUSSION TITLE
error while loading shared library

### DISCUSSION THREAD

**Discussion by Emmanuel-324** [idaholab/moose#26090] [Q&A General]
i was trying to run the step1.i file in MOOSE repository and below is the error message I get.

[emmanuel324@tinkercliffs1 problems]$ ../darcy_thermo_mech-opt -i step1.i
../darcy_thermo_mech-opt: error while loading shared libraries: libcombined-opt.so.0: cannot open shared object file: No such file or directory

**Comment by GiudGiud** [comment]
Hello

You should clean the repo then recompile.
The executable was compiled (and linked to) with that library but then it was deleted seemingly

Guillaume
